src/program.cpp

```diff
@@ -896,9 +896,10 @@ cl_build_status cvk_program::link() {
 
     m_binary.use(std::move(linked_opt));
 
-    // Merge descriptor maps
-    for (cl_uint i = 0; i < m_num_input_programs; i++) {
-        m_binary.insert_descriptor_map(m_input_programs[i]->m_binary);
```

Comment:
This was the last use of `spir_binary::insert_descriptor_map`. We can delete the function entirely as well as `spir_binary::m_dmaps_text`. Good riddance :).